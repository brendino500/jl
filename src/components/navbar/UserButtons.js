import { makeStyles } from '@material-ui/core/styles'
import { Button } from '@material-ui/core'

const useStyles = makeStyles((theme) => ({
  buttonText: {
    color: 'black',
    fontFamily: 'Sora',
    marginLeft: '1em',
  },
}))

export default function UserButtons({ text }) {
  const classes = useStyles()
  return (
    <>
      <Button
        disableRipple="true"
        className={classes.buttonText}
        href="http://square-enix-games.com"
        target="_blank"
        rel="noreferrer"
      >
        {text}
      </Button>
    </>
  )
}
