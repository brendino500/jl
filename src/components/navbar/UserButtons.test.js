import renderer from 'react-test-renderer'
import UserButtons from './UserButtons'

it('renders correctly', () => {
  const tree = renderer.create(<UserButtons />).toJSON()
  expect(tree).toMatchSnapshot()
})
