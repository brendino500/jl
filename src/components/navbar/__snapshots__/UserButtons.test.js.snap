// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
<a
  aria-disabled={false}
  className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-buttonText-1"
  href="http://square-enix-games.com"
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  rel="noreferrer"
  tabIndex={0}
  target="_blank"
>
  <span
    className="MuiButton-label"
  />
</a>
`;

exports[`it renders correctly 1`] = `
<a
  aria-disabled={false}
  className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-buttonText-1"
  href="http://square-enix-games.com"
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  rel="noreferrer"
  tabIndex={0}
  target="_blank"
>
  <span
    className="MuiButton-label"
  />
</a>
`;

exports[`renders correctly 1`] = `
<a
  aria-disabled={false}
  className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-buttonText-1"
  href="http://square-enix-games.com"
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  rel="noreferrer"
  tabIndex={0}
  target="_blank"
>
  <span
    className="MuiButton-label"
  />
</a>
`;
